cohort_4/playbook/schedule.md

```diff
@@ -1,5 +1,9 @@
 # Schedule
 
+## Attendance
+
+Hacker attendance should be tracked [here](https://docs.google.com/spreadsheets/d/1Em5tUQdm4dXnthtWr4zY4KR9Q6JZMV_Hl2n6G9Ti9Bg/edit)
+
```

Comment:
Now that we have the "Hackers arrive" section in `activities.md`, do we still need this section?